src/lib.rs

```diff
@@ -58,16 +59,45 @@ use iron::method::Method;
 use iron::status;
 use iron::headers;
 
+/// The struct that builds a CorsMiddleware
+pub struct CorsMiddlewareBuilder {
+    allowed_hosts: Option<HashSet<String>>,
+    allow_credentials: bool
+}
+
+impl CorsMiddlewareBuilder {
+    pub fn new() -> Self {
+        CorsMiddlewareBuilder { allow_credentials: false, allowed_hosts: None }
+    }
+
+    /// Specify which origin hosts are allowed to access the resource.
+    pub fn allowed_hosts(&mut self, allowed_hosts: HashSet<String>) -> &mut Self {
+        self.allowed_hosts = Some(allowed_hosts);
+        self
+    }
+    /// Specify Access-Control-Allow-Credentials
```

Comment:
Can you document the default value?

    /// By default, the `AccessControlAllowCredentials` header will be set to `false`.